src/DynamoCore/Search/SearchElements/NodeSearchElement.cs

```diff
@@ -29,6 +29,7 @@ private void OnVisibilityChanged()
             if (VisibilityChanged != null)
                 VisibilityChanged();
         }
+        internal int PortToConnect { get; set; }
```

Comment:
this seems like an inappropriate place for this data

Reply:
I kinda agree, if needed, we may want to inherit this class. I think in the future we may add more properties if need, e.g. favorite, weight, confidence level (after ML kicks in).

Reply:
Since this data will be specific to each search element in the auto-completion window, do we want to create a new class for these node elements and that would inherit the NodeSearchElement class?

Reply:
@reddyashish Yeah, I would be supportive